run-windows.cmd

```diff
@@ -1,4 +1,5 @@
 @echo off
-start .\bin\cosim-demo-app.exe
+cd bin
+start cosim-demo-app.exe
```

Comment:
Seems like working directory must be the same as path to cosim-demo-app executable for libcosim to find proxyfmu.

Reply:
We may want to add some clever logic to work around this in libcosim, such as looking for the proxy-fmu server executable in the same directory as the main program executable. We have [some code in Coral](https://github.com/viproma/coral/blob/master/src/lib/util.cpp#L253) that could be put to such a use (and it is not unlikely that Boost.Process has gained similar functionality since I wrote that).

Reply:
Created a [libcosim issue](https://github.com/open-simulation-platform/libcosim/issues/653)